### PR TITLE
add deploy with dapp docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

Adds a page on deploying an appchain with the tanssi dapp. I feel like this should be more technical than it is, but I don't know if it is just the nature of using the dapp or if I can expand on some of the details more. Thinking probably a bit of the latter so let me know! 🙂

### CODE REVIEW

**Review by albertov19 — APPROVED** [review]
LGTM

I'm not a fan of the `/builders/deploy/dapp/deploy/` flow: maybe we can change the first `deploy` to `deploy-manage`?

File: builders/deploy/dapp/deploy.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+Once the transaction has successfully gone through, your Appchain ID will be displayed on the dApp, and you'll be able to click **Continue** to proceed to the next step. You'll notice that on your Alphanet relay chain account, 20 UNIT tokens have been removed from your transferrable balance and are now reserved.
+
+![Successfully reserved your Appchain ID via the Tanssi dApp.](/images/builders/deploy/dapp/deploy/deploy-8.png)
+
+## Generate Your Appchain Files {: #generate-appchain-files }
+
+Before you can deploy your Appchain, you'll need to generate four configuration files: 
+
+1. The raw chain specification
+2. The genesis state
```
Comment (albertov19) [inline builders/deploy/dapp/deploy.md:145]: Add some brief info on what each of the files is? (cc @augustocollado can help)

Comment (augustocollado) [reply builders/deploy/dapp/deploy.md:145]: We can add links to the chain specs articles instead:
/builders/build/substrate/customizing-chain-specs/#generating-raw-specs-file
/builders/build/substrate/customizing-chain-specs/#genesis-state

**Review by augustocollado — APPROVED** [review]
Great work!
I'd be aware that we refer to Appchain when developing and preparing to launch, and ContainerChain when it is deployed & alive, so at least in the last section I'd change the title to "Deploy your ContainerChain"

File: builders/deploy/dapp/deploy.md (hunk 1 above)
Comment (eshaben) [reply builders/deploy/dapp/deploy.md:145]: what link for genesis wasm? can i use same link for genesis state and tanssi genesis state? @augustocollado

Comment (eshaben) [reply builders/deploy/dapp/deploy.md]: so what would i do for the wasm file?

Comment (augustocollado) [reply builders/deploy/dapp/deploy.md]: for wasm, we can use /learn/framework/architecture/#runtime 
Tanssi genesis state....not sure, let me check!

Comment (augustocollado) [reply builders/deploy/dapp/deploy.md]: For tanssi, same link as genesis state

File: builders/deploy-manage/dapp/deploy.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+![Successfully reserved your Appchain ID via the Tanssi dApp.](/images/builders/deploy-manage/dapp/deploy/deploy-8.png)
+
+## Generate Your Appchain Files {: #generate-appchain-files }
+
+Before you can deploy your Appchain, you'll need to generate four configuration files:
+
+- [The raw chain specification](/builders/build/substrate/customizing-chain-specs/#generating-raw-specs-file){target=_blank} - a compact version of the JSON specification file, which defines the initial settings and state that all nodes participating in the network must agree on to reach consensus and produce blocks
+- [The genesis state](/builders/build/substrate/customizing-chain-specs/#genesis-state){target=_blank} - defines the initial state upon which all transactions and state transitions are executed
+- The genesis Wasm - a WebAssembly (Wasm) blob that defines the runtime validation logic
+- The Tanssi genesis state - defines the genesis state for Tanssi
```
Comment (eshaben) [inline builders/deploy-manage/dapp/deploy.md:147]: @augustocollado can you please review this part specifically?

**Comment by albertov19** [comment]
> Great work! I'd be aware that we refer to Appchain when developing and preparing to launch, and ContainerChain when it is deployed & alive, so at least in the last section I'd change the title to "Deploy your ContainerChain"

I don't fully agree with this, because what you are deploying is actually an Appchain. It becomes a ContainerChain after it has deployed right?

File: builders/deploy-manage/dapp/deploy.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+![Successfully reserved your Appchain ID via the Tanssi dApp.](/images/builders/deploy-manage/dapp/deploy/deploy-8.png)
+
+## Generate Your Appchain Files {: #generate-appchain-files }
+
+Before you can deploy your Appchain, you'll need to generate four configuration files:
+
+- [The raw chain specification](/builders/build/substrate/customizing-chain-specs/#generating-raw-specs-file){target=_blank} - a compact version of the JSON specification file, which defines the initial settings and state that all nodes participating in the network must agree on to reach consensus and produce blocks
+- [The genesis state](/builders/build/substrate/customizing-chain-specs/#genesis-state){target=_blank} - defines the initial state upon which all transactions and state transitions are executed
+- [The genesis Wasm](/learn/framework/architecture/#runtime){target=_blank} - a WebAssembly (Wasm) blob that defines the runtime validation logic
+- [The Tanssi genesis state](/builders/build/substrate/customizing-chain-specs/#genesis-state){target=_blank} - defines the genesis state for Tanssi
```
Comment (eshaben) [inline builders/deploy-manage/dapp/deploy.md:147]: @augustocollado added a brief description of each, please review this specifically

Comment (augustocollado) [reply builders/deploy-manage/dapp/deploy.md:147]: - a WebAssembly (Wasm) blob that defines the runtime validation logic -> a WebAssembly (Wasm) blob that defines the runtime logic
- defines the genesis state for Tanssi -> defines the genesis state to register in Tanssi

**Review by augustocollado — APPROVED** [review]
lgtm!

**Pull request merged**